packages/integration-sdk-dev-tools/config/eslint.json

```diff
@@ -47,6 +47,7 @@
   ],
   "rules": {
     "no-unused-vars": "off",
+    "no-console": "error",
```

Comment:
Have you confirmed (perhaps by linking `@jupiterone/integration-sdk-dev-tools` in a local version of `graph-aws`) that the addition of this rule causes `yarn lint` to fail if you add a `console.log()` statement somewhere in `graph-aws`?

Reply:
<img width="645" alt="Screen Shot 2022-09-21 at 10 21 54 AM" src="https://user-images.githubusercontent.com/89530949/191530171-7f27548a-763e-4d71-a16e-9d2cf16f458d.png">

Reply:
Just checked  and yes it does seem to catch console.log